app/views/shared/_in-person-verification-results-email-lower.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="mt-40">
+  <table class="button expanded large radius">
+    <tbody>
+      <tr>
+        <td>
+          <table>
+            <tbody>
+              <tr>
+                <td style="text-align: center">
+                  <%= link_to t('user_mailer.in_person_verified.sign_in'),
+                              idv_url,
+                              target: '_blank',
+                              class: 'float-center',
+                              rel: 'noopener' %>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </td>
+        <td class="expander"></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<p>
+  <%= link_to idv_url, idv_url, target: '_blank', rel: 'noopener' %>
+</p>
+
+<div class="s16">
```

Comment:
Separately we should consider changing the default font size for all emails, so we're not applying them to individual templates like this.